metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_config.py

```diff
@@ -326,6 +327,18 @@ class SnowflakeV2Config(
         " Map of share name -> details of share.",
     )
 
+    known_snowflake_edition: Optional[SnowflakeEdition] = Field(
+        default=None,
+        description="Explicitly specify the Snowflake edition (STANDARD or ENTERPRISE). If unset, the edition will be inferred automatically using 'SHOW TAGS'.",
+    )
+
+    # Allows empty containers to be ingested before datasets are added, avoiding permission errors
+    warn_no_datasets: bool = Field(
+        hidden_from_docs=True,
+        default=True,
```

Comment:
actually the default should be false here right? that would maintain the existing behavior as the default

```suggestion
        default=False,
```

Reply:
Right, changed default to `False`